docs/sql-ref-syntax-aux-describe-database.md

```diff
@@ -18,5 +18,79 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+### Description
+​
+`DESCRIBE DATABASE` statement returns the metadata of an existing database. The metadata information includes database
+name, database comment, and database location on the filesystem. If the optional `EXTENDED` option is specified, it
+returns the basic metadata information along with the database properties. The `DATABASE` and `SCHEMA` are
+interchangeable.
 
-**This page is under construction**
+### Syntax
+{% highlight sql %}
+{DESC | DESCRIBE} DATABASE [EXTENDED] db_name
+{% endhighlight %}
+
+### Parameters
+<dl>
+  <dt><code><em>db_name</em></code></dt>
+  <dd>
+    Specifies a name of an existing database or an existing schema in the syetem. If the name does not exist, an
+    exception is thrown. The name is case insensitive, it is stored as low case in SPARK<br><br>
+  </dd>
+</dl>
+
+### Example
+{% highlight sql %}
+-- Create employees DATABASE
+CREATE DATABASE EMPLOYEES COMMENT 'For software companies';
```

Comment:
@kevinyu98 lets use lowercase for `EMPLOYEES` to match other PRS..

Reply:
@dilipbiswal sure